robyn/__init__.py

```diff
@@ -17,29 +27,34 @@ def start(self, port):
     def get(self, endpoint):
         def inner(handler):
             self.add_route("GET", endpoint, handler)
+
         return inner
-    
+
     def post(self, endpoint):
         def inner(handler):
+            sig = signature(handler)
+            params = len(sig.parameters)
+            if params != 1:
+                print("We need one argument on post.")
```

Comment:
@JackThomson2 , I think we should raise an exception here instead of a print.

Reply:
Probably

Reply:
Giving it another thought, print should be better otherwise we would have to handle exceptions manually.